skgstat/interfaces/gstools.py

```diff
@@ -97,3 +97,125 @@ def skgstat_to_gstools(variogram, **kwargs):
+    unbiased : :class:`bool`, optional
+        Whether the kriging weights should sum up to 1, so the estimator
+        is unbiased. If unbiased is `False` and no drifts are given,
+        this results in simple kriging.
+        Default: True
+    exact : :class:`bool`, optional
+        Whether the interpolator should reproduce the exact input values.
+        If `False`, `cond_err` is interpreted as measurement error
+        at the conditioning points and the result will be more smooth.
+        Default: False
+    cond_err : :class:`str`, :class :class:`float` or :class:`list`, optional
+        The measurement error at the conditioning points.
+        Either "nugget" to apply the model-nugget, a single value applied to
+        all points or an array with individual values for each point.
+        The "exact=True" variant only works with "cond_err='nugget'".
+        Default: "nugget"
+    pseudo_inv : :class:`bool`, optional
+        Whether the kriging system is solved with the pseudo inverted
+        kriging matrix. If `True`, this leads to more numerical stability
+        and redundant points are averaged. But it can take more time.
+        Default: True
+    pseudo_inv_type : :class:`str` or :any:`callable`, optional
+        Here you can select the algorithm to compute the pseudo-inverse matrix:
+
+            * `"pinv"`: use `pinv` from `scipy` which uses `lstsq`
+            * `"pinv2"`: use `pinv2` from `scipy` which uses `SVD`
+            * `"pinvh"`: use `pinvh` from `scipy` which uses eigen-values
+
+        If you want to use another routine to invert the kriging matrix,
+        you can pass a callable which takes a matrix and returns the inverse.
+        Default: `"pinv"`
+    fit_normalizer : :class:`bool`, optional
+        Wheater to fit the data-normalizer to the given conditioning data.
+        Default: False
+
+    Raises
+    ------
+    ImportError
+        When GSTools is not installed.
+    ValueError
+        When GSTools version is not v1.3 or greater.
+    ValueError
+        When given Variogram model is not supported ('harmonize').
+
+    Returns
+    -------
+    :any:`Krige`
+        Instantiated GSTools Krige class.
+
+    Note
+    ----
+    The documentation for the keyword Arguments is directly
+    taken from gstools==1.3.0 documentation. If you are running
+    a more recent version, the arguments might differ.
+
+    """
+    # try to import gstools and notify user if not installed
+    try:
+        import gstools as gs
+    except ImportError as e:
+        raise ImportError("to_gstools: GSTools not installed.") from e
+
+    # at least gstools>=1.3.0 is needed
+    if list(map(int, gs.__version__.split(".")[:2])) < [1, 3]:
+        raise ValueError("to_gstools: GSTools v1.3 or greater requiered.")
+
+    # convert variogram to a CovModel
+    model = skgstat_to_gstools(variogram=variogram)
+
+    # extract cond_pos and cond_vals
+    cond_pos = list(zip(*variogram.coordinates))
```

Comment:
when `variogram.coordinates` is a numpy array, I would argue for `variogram.coordinates.T` since this is faster.

Reply:
agree